tests/http/clients/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ foreach(_test_name IN LISTS check_PROGRAMS)
   add_executable(${_test_target} EXCLUDE_FROM_ALL "${_test_name}.c")
   add_dependencies(testdeps ${_test_target})
   target_include_directories(${_test_target} PRIVATE
+    "${CURL_SOURCE_DIR}/include"  # for "curl/curl.h" etc
```

Comment:
The include directory is added globally here:
https://github.com/curl/curl/blob/9724526429bb5c41fc7b923c616e4c466faf5f7a/CMakeLists.txt#L113

Reply:
that was clearly just a desperate attempt, will clean up once the format PR has landed.